submission/races_test.go

```diff
@@ -33,20 +33,13 @@ func testdataSCT() *ct.SignedCertificateTimestamp {
 	return &sct
 }
 
-// mockSubmitter keeps track of number of requests per log-group. Logs split into groups based on logURL first letter.
 type mockSubmitter struct {
-	firstLetterURLReqNumber map[byte]int
-	mu                      sync.Mutex
 }
 
 // Each request within same Log-group gets additional sleep period.
-func (ms *mockSubmitter) SubmitToLog(_ context.Context, logURL string, _ []ct.ASN1Cert, _ bool) (*ct.SignedCertificateTimestamp, error) {
-	ms.mu.Lock()
-	reqNum := ms.firstLetterURLReqNumber[logURL[0]]
-	ms.firstLetterURLReqNumber[logURL[0]]++
-	ms.mu.Unlock()
```

Comment:
Is this change to the test file related to the bug fix?  I'm struggling to understand the motivation behind it.  I should mention that I'm new to the code for this particular tool, so am learning as I go, so feel free to just explain why it's being done :)